TomlUtils.cs

```diff
@@ -67,5 +68,27 @@ public static string SafegetString(TomlNode node, string key)
                 return null;
             }
         }
+
+        public static string[] SafeGetArray(TomlNode node, string key)
+        {
+            try
+            {
+                return node[key].AsArray.RawArray.Select(x => x.AsString.Value).ToArray();
+            }
+            catch (NullReferenceException)
+            {
+                return Array.Empty<string>();

```

Comment:
I feel like this should be null to stay in line with the others. Also it's a more sane default for "not found".

Since it's also practical to have an empty array be returned, I'd suggest you add an optional `defaultValue` param which will be returned (if set) instead of null.